core/src/main/scala/org/apache/spark/SparkStatusTracker.scala

```diff
@@ -104,4 +106,34 @@ class SparkStatusTracker private[spark] (sc: SparkContext) {
       }
     }
   }
+
+  /**
+   * Returns a list of all known executors, represented by string with format: "host:port"
+   */
+  def getExecutors(): Array[String] = {
```

Comment:
wait, how is `getExecutorList` different from `getExecutors`? Why not just be more specific what the strings are?